htdocs/societe/list.php

```diff
@@ -289,8 +297,10 @@
 		$search_id = '';
 		$search_nom = '';
 		$search_alias = '';
-		$search_categ_cus = 0;
-		$search_categ_sup = 0;
+		$searchCategoryCustomerList = array();
```

Comment:
For consistency of code and to be ready for the future generic filter, can you keep name of each var used to store a filter on select with syntax $search_xxxx ?